packages/block/test/eip7685block.spec.ts

```diff
@@ -1,14 +1,17 @@
 import { Common, Hardfork, Mainnet } from '@ethereumjs/common'
```

Comment:
I'd say most of the tests in [eip7685block.spec.ts](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3706/files#diff-4a12d82d034dcf55abb6f0827f9daa3e58807f5b20457e3a144fa88296901f41) should be removed since they are build on the old spec version which still included requests in the execution block body. Since the new version of the eips and spec now does not included the requests in the block body, these tests will have to be rewritten to properly check the validity of the requests on the execution-client side.